remotespark/livyclientlib/pandaspysparklivyclient.py

```diff
@@ -24,9 +24,10 @@ def get_columns_dataframe(self, columns_text):
         records = list()
```

Comment:
I forgot to incorporate this rename into my last commit. I can go ahead and do that as well